less/roxie/js/custom.js

```diff
@@ -43,15 +43,15 @@ $(window).scroll(function(){
 //TWITTER SHARE BUTTON
 //!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src="https://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");
 
-$(document).ready(function() {	
+$(document).ready(function() {
 	//NAVBAR
-	$('.navbar-main-menu > li.dropdown').mouseenter(function() {
-		$(this).addClass('open');
-	});
-	
-	$('.navbar-main-menu > li.dropdown').mouseleave(function() {
-		$(this).removeClass('open');
-	});
+	//$('.navbar-main-menu > li.dropdown').mouseenter(function() {
```

Comment:
In that case, I will remove everything that is currently commented out because I agree.